#165965800 remove user data on signup response

ericnyirimana opened the pull request:
Description
=======
We need to refactor the response got when signing up by removing user data information for privacy purpose. We just expect the response to have the user token and a message for successful registration.

Type of change
=======
- Removed user data on the response in the controller
- change expected responses on user signup test

How has it been tested
=======
- It is tested when you signup via ```api/v1/users/signup```

Checklist:
=======
N/A

Pivotal tracker story ID
=======
[165965800](https://www.pivotaltracker.com/n/projects/2326984/stories/165965800)

ericnyirimana committed:
chore(authentication): remove user data on signup response
- Removed user data on response in the controller
- change expected response on user signup test
[Delivers #165965800]

Niyitangasam reviewed (approved):
That's fine as per PO feedback.

ericnyirimana committed:
chore(authentication): remove expected response on signup
- Removed expected responses on rating test when signing up
[Delivers #165965800]
